content/en/docs/tasks/network/customize-hosts-file-for-pods.md

```diff
@@ -118,9 +118,9 @@ with the additional entries specified at the bottom.
 ## Why does the kubelet manage the hosts file? {#why-does-kubelet-manage-the-hosts-file}
 
 The kubelet [manages](https://github.com/kubernetes/kubernetes/issues/14633) the
-`hosts` file for each container of the Pod to prevent Docker from
-[modifying](https://github.com/moby/moby/issues/17190) the file after the
-containers have already been started.
+`hosts` file for each container of the Pod to prevent the container runtime from
+modifying the file after the containers have already been started.
+For e.g. earlier Docker used to [modify](https://github.com/moby/moby/issues/17190) the `hosts` file after the containers start.
```

Comment:
How about:
```suggestion
Historically, Kubernetes always used Docker Engine as its container runtime, and Docker Engine would
then modify the `/etc/hosts` file after each container had started.

Current Kubernetes can use a variety of container runtimes; even so, the kubelet manages the
hosts file within each container so that the outcome is as intended regardless of which
container runtime you use.
```

Reply:
Yes, this seems fair to write as we currently don't have any evidence on whether other container runtimes have also tried to modify the hosts file.
Should we link the PR about the Docker problem though for better context?

Reply:
The pages we're linking to are old. It's fine to link to something current but in my view those are best seen as stale.

Reply:
Okay. Thanks for the suggestion, making the change!